src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -379,16 +907,37 @@
 		"sourcePath" : "log.sagebase.org/certifiedUserPassingRecords/",
 		"tableDescriptor": {
 			"name": "certifiedquizquestionrecords",
-			"columns": {
-				"response_id": "bigint",
-				"question_index": "bigint",
-				"is_correct": "boolean",
-				"stack": "string",
-				"instance": "string"
+			"description" : "This table contains records of the questions of certified quiz taken bu Synapse user.",
+			"columns":[{
+				"name": "response_id",
+				"type" : "bigint",
+				"comment" : "The identifier of the certified quiz record associated with this question record."
```

Comment:
I don't understand this description at all.

Can we use the phrase "certification quiz" rather than "certified quiz". It's the user who is certified, not the quiz.